worldedit_commands/cuboid.lua

```diff
@@ -28,18 +30,18 @@ worldedit.register_command("outset", {
 			assert(worldedit.cuboid_linear_expand(name, 'y', 1, amount))
 			assert(worldedit.cuboid_linear_expand(name, 'y', -1, amount))
 		else
-			return false, "Invalid number of arguments"
+			return false, S("Invalid number of arguments")
 		end
 
 		worldedit.marker_update(name)
-		return true, "Region outset by " .. amount .. " blocks"
+		return true, S("Region outset by @1 blocks", amount)
```

Comment:
sorry to ask for more work but if this is being touched anyway can you fix the terminology?
"blocks" is wrong and should be "nodes"

Reply:
yep, also fixed for `//inset` command